packages/components/organisms/f-checkout/src/store/checkoutExperimentation.module.js

```diff
@@ -0,0 +1,36 @@
+import { HEADER_LOW_VALUE_ORDER_EXPERIMENT } from '../constants';
+
+import {
+    UPDATE_EXPERIMENTS_STATE
+} from './mutation-types';
+
+export default {
+    namespaced: true,
+
+    state: () => ({
+        lowValueOrderExperimentValue: ''
```

Comment:
I'm wondering if having "experiment value" on the end of the variable names is necessary, I think they're probably both implied?

Reply:
I hijacked this from menuweb but happy to remove the suffixes.

Reply:
We use an `isAbcExperimentActive` convention on menu which reads nicely in components.

Reply:
That is actually correct. I must have forgot what part I copied 😂

Reply:
I'm gonna change it to `lowValueOrderExperimentVariant`, as that's what it seems to be being used as